dpdata/plugins/lammps.py

```diff
@@ -45,7 +62,18 @@ def to_system(self, data, file_name: FileType, frame_idx=0, **kwargs):
 class LAMMPSDumpFormat(Format):
     @Format.post("shift_orig_zero")
     def from_system(
-        self, file_name, type_map=None, begin=0, step=1, unwrap=False, **kwargs
+        self,
+        file_name,
+        type_map=None,
+        begin=0,
+        step=1,
+        unwrap=False,
+        input_name=None,
```

Comment:
I am wondering if it is over-designed to read the input file.

Reply:
When we want to read spin information from the dump file, we need to search for keywords related to spin information based on the settings of the input file